tcmu-runner.spec

```diff
@@ -1,3 +1,29 @@
+%global _hardened_build 1
+
+# uncomment and add '%' to use the prereltag for pre-releases
+# %%global prereltag qa3
```

Comment:
where are we using prereltag ?

I understand you want to use prereltag at 'Release:' and/or with 'Source:', but I don't see its usage in this file.

Reply:
Done, removed.